packages/ember-views/lib/mixins/view_support.js

```diff
@@ -70,7 +70,6 @@ export default Mixin.create({
   concatenatedProperties: ['attributeBindings'],
 
   [POST_INIT]() {
-    this.trigger('didInitAttrs');
```

Comment:
I think this needs to be `ENV` wrapped, otherwise users in the addon will not ever have their `didInitAttrs` get invoked...

Reply:
@rwjblue Oops forgot about that. You are right